app/pages/cif/project-revision/[projectRevision]/view.tsx

```diff
@@ -15,8 +15,17 @@ import readOnlyTheme from "lib/theme/ReadOnlyTheme";
 import { getLocaleFormattedDate } from "lib/theme/getLocaleFormattedDate";
 import useShowGrowthbookFeature from "lib/growthbookWrapper";
 import NotifyModal from "components/ProjectRevision/NotifyModal";
+import { useUpdateProjectRevision } from "mutations/ProjectRevision/updateProjectRevision";
+import { Button } from "@button-inc/bcgov-theme";
+import ReadOnlyWidget from "lib/theme/widgets/ReadOnlyWidget";
+import { WidgetProps } from "@rjsf/core";
+import SelectWidget from "lib/theme/widgets/SelectWidget";
+import { useState } from "react";
 
-const createProjectRevisionViewSchema = (allRevisionTypes) => {
+const createProjectRevisionViewSchema = (
+  allRevisionTypes,
+  allRevisionStatuses
```

Comment:
the two parameters are inconsistent (we should type them to avoid it and increase readability)
- `allRevisionTypes` is an object `{ edges: { node: ... }[]}`
- `allRevisionStatuses` is an array of edges

I suggest making them either both an array of edges, and calling them `allRevisionTypesEdges` and `allRevisionStatusesEdges`